infrastructure/swagger/05_paths.yaml

```diff
@@ -285,6 +285,46 @@ paths:
     #     - app-level0: []
 
   /ProductTeam/{product_team_id}/Product/{product_id}:
+    options:
```

Comment:
We need to add a "tag" into all options as they currently just display under "default" category